src/components/content-tab-import-export.tsx

```diff
@@ -22,6 +23,10 @@ interface ContentTabImportExportProps {
 export const ExportSite = ( { selectedSite }: { selectedSite: SiteDetails } ) => {
 	const { exportState, exportFullSite, exportDatabase } = useImportExport();
 	const { [ selectedSite.id ]: currentProgress } = exportState;
+	const isSiteImporting = selectedSite?.importState === 'importing';
+	const importProgressMessage = __(
+		'This site is being imported. Please wait for the import to finish.'
```

Comment:
This is displayed on the export button, but mentions only the import. Should we mention export to explain the context?

```suggestion
		'This site is being imported. Please wait for the import to finish before you export it.'
```